pkg/ring/lifecycler.go

```diff
@@ -50,7 +50,7 @@ type LifecyclerConfig struct {
 	JoinAfter        time.Duration `yaml:"join_after,omitempty"`
 	MinReadyDuration time.Duration `yaml:"min_ready_duration,omitempty"`
 	UnusedFlag       bool          `yaml:"claim_on_rollout,omitempty"` // DEPRECATED - left for backwards-compatibility
```

Comment:
Can we move the unused flags to bottom of the struct definition?

Reply:
Done.